test/functional/configuration/configuration.spec.js

```diff
@@ -286,6 +286,8 @@ test("returns full valid settings", async () => {
   await instances[1].idMigrationEnabled.click();
   await instances[1].thirdPartyCookiesEnabled.click();
   await instances[1].onBeforeEventSendEditButton.click();
+  // Click on the field before clearing to get rid of the "..."
+  await instances[1].downloadLinkQualifierField.click();
```

Comment:
I suspect moving to parcel v2 introduced a testCafe timing issue when clearing long textfields.